<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Alexander Meinke</author>
 <class>QBluetooth</class>
 <widget class="QMainWindow" name="QBluetooth">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QBluetooth</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Device">
       <attribute name="title">
        <string>Device</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0" rowspan="6">
         <widget class="QListWidget" name="listWidget"/>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="toolButtonConnect">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="toolButtonScan">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Scan</string>
          </property>
          <property name="icon">
           <iconset resource="qbluetooth.qrc">
            <normaloff>:/images/images/network-wireless.png</normaloff>:/images/images/network-wireless.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Display">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Display</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" rowspan="5">
         <widget class="QGraphicsView" name="graphicsView">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="backgroundBrush">
           <brush brushstyle="NoBrush">
            <color alpha="64">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
          <property name="sceneRect">
           <rectf>
            <x>0.000000000000000</x>
            <y>0.000000000000000</y>
            <width>128.000000000000000</width>
            <height>64.000000000000000</height>
           </rectf>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="resizeAnchor">
           <enum>QGraphicsView::NoAnchor</enum>
          </property>
          <property name="viewportUpdateMode">
           <enum>QGraphicsView::MinimalViewportUpdate</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="toolButtonSend">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QToolButton" name="toolButtonClear">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="verticalSliderZoom">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarDevice">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarDisplay">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qbluetooth.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>QBluetooth</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
